backend/clinical_trials.py

```
@@ -157,6 +157,86 @@ def _sort_key(t: Dict):
+        payload = resp.json()
+    except Exception as e:
+        print(f"   ⚠️  CT.gov modality search error ({condition} / {modality}): {e}")
+        return []
+
+    studies = payload.get("studies") or []
+    trials: List[Dict] = []
+    for raw in studies:
+        try:
+            trials.append(parse_trial(raw))
+        except Exception:
+            continue
```

Ruff (0.15.12)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)